chsdi/templates/htmlpopup/oev_haltestellen.mako

```diff
@@ -0,0 +1,151 @@
+<%inherit file="base.mako"/>
+
+<%def name="table_body(c, lang)">
+    <%!
+        from pyramid.url import route_url
```

Comment:
yes, because I need to add the link for the extended tooltip and I add this manually, because I must change the style of the mako.